app/helpers/application_form_helper.rb

```diff
@@ -108,4 +108,14 @@ def application_form_summary_rows(
       { key: { text: key }, value: { text: value }, actions: actions || [] }
     end
   end
+
+  def display_banner?(application_form)
```

Comment:
Could we clarify the name of this? Something that explains which banner is being displayed would be ideal.